app/controllers/my-sessions/view.js

```diff
@@ -13,6 +13,15 @@ export default class extends Controller {
 
     return true;
   }
+  @computed('model.event.speakersCall.endsAt')
+  get isCfsOpen() {
+    let cfsEndsAt = this.get('model.event.speakersCall.endsAt');
+    if (cfsEndsAt  < moment()) {
+      return false;
```

Comment:
```js
return ((this.model.event.speakersCall.endsAt < moment()) ? true : false )

Reply:
Not working , i don't know why . it is supposed to work . it should be
` return ((this.model.event.speakersCall.endsAt < moment()) ? false : true )` , but that's not working too .
Just rendering this empty page -
![Screenshot from 2020-01-07 00-45-26](https://user-images.githubusercontent.com/56407566/71842352-6fd48a00-30e7-11ea-8293-2059af3fdec1.png)

Reply:
```js
return this.model.event.speakersCall.isOpen
```
Why compare the time when there is a computed property already present? Use it and change correspondingly.

Reply:
instead of making a computed function i used :
`{{#if model.event.speakersCall.isOpen}}`
and this is working .
![Screenshot from 2020-01-07 14-14-06](https://user-images.githubusercontent.com/56407566/71881262-fe80ef80-3157-11ea-8612-34abc123e51c.png)

Reply:
> ```js
> return this.model.event.speakersCall.isOpen
> ```
> 
> Why compare the time when there is a computed property already present? Use it and change correspondingly.
tried this too but not working -
```
@computed('model.event.speakersCall.endsAt')
get isCfsOpen(){
    return this.model.event.speakersCall.isOpen;
    }
```

Reply:
Why to make a computed property now, When there is a field which checks the availability for you ?
```js
{{#if model.event.speakersCall.isOpen}}
    Code
{{/if}}
```

Reply:
> Why to make a computed property now, When there is a field which checks the availability for you ?
> 
> ```js
> {{#if model.event.speakersCall.isOpen}}
>     Code
> {{/if}}
> ```

exactly